Fix broken imports

jkonecny12 opened the pull request:
Fix import broken by recent Anaconda refactoring.
Fix addons must use simpleline a new library.
Fix code style.

vpodzime commented on org_fedora_hello_world/tui/spokes/hello_world.py:
```diff
@@ -198,84 +193,20 @@ def input(self, args, key):
-        Entry("Always shown input", "shown_input", _valid_input, True),
-        Entry("Conditioned input", "hidden_input", _valid_input,
-              lambda self, args: bool(args.shown_input)),
-        ]
```
I believe we still need an example of such "inputs".

jkonecny12 replied:
Ok, then I'll add them back but I need to do it with Simpleline.
Right now kdump addon has a priority so I'll fix this later.

jkonecny12 replied:
On the other hands this will be changed dramatically because of the Anaconda modularization effort so I think we would need to change the hello world later even more. In this case I don't see a point in improving it right now. 

There's a link on the Simpleline documentation where is this described already. I know it is not that comfortable but it should be enough.

vpodzime replied:
> Ok, then I'll add them back but I need to do it with Simpleline.

People having their own addons or writing new addons will have to to this too. Don't take me wrong, but if it's really that hard for you as the author of Simpleline, how can you expect people being able to do it?

jkonecny12 replied:
It's not that hard but it is unpleasant and time consuming to do it in Anaconda by updates image. And yes, Simpleline still has reserve in comfort of usage and I hope I'll be able to fix these soon. I'll already started. One new simplification is on it's way and another is in TODO list. BTW I would welcome any ideas for improvement as issues in repo or bugs in bugzilla just in case you have some.

Ok, I guess I can take example solution and just polite it, then it shouldn't take much time.

jkonecny12 commented:
**UPDATED:**
* Added `HelloWorldEditSpoke`

I've added the `HelloWorldEditSpoke` but it will be changed soon so or so. First because of changes in Simpleline I want to have there and second because the whole Anaconda is changing.
